NEWS

```diff
@@ -1,3 +1,10 @@
+shiny-server 1.5.2
+--------------------------------------------------------------------------------
+
+* Add additional configuration directives `http_keepalive_timeout`,
+  `sockjs_heartbeat_delay`, and `sockjs_disconnect_delay` to allow working
+  with very slow connections and large SockJS payloads.
+
```

Comment:
Might be worth clarifying whether or not any of these default values changed from the previous release. (I'm curious)